### PR TITLE
Added asyncio.queues stubs

### PR DESCRIPTION
**Pull request by ze-phyr-us**
Just a few basic stubs - I'm quite new to this so happy to improve if you have any suggestions.

I've also fixed the construction of `__all__` in `stubs/3.4/asyncio/__init__.pyi`.

### CODE REVIEW

File: stubs/3.4/asyncio/queues.pyi [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from asyncio.events import AbstractEventLoop
+from .tasks import coroutine
+
+
+class QueueEmpty(Exception): ...
+class QueueFull(Exception): ...
+
+
+class Queue:
+    def __init__(self, maxsize: int = 0, *, loop: AbstractEventLoop = None): ...
```
Comment (spkersten) [inline stubs/3.4/asyncio/queues.pyi]: The return type of `__init__` should be `None`.

**Comment by ze-phyr-us** [comment]
I've added the `__init__` return type and a type parameter.

`JoinableQueue` defines `join` and `task_done`, not `Queue`.

**Comment by JukkaL** [comment]
Thanks for the PR! I'm currently busy traveling in Europe but I'll be back to this as soon as I'm back home next week.

**Comment by JukkaL** [comment]
Apparently in Python 3.4.4, `JoinableQueue` will be an alias for `Queue` (https://docs.python.org/3/library/asyncio-queue.html#joinablequeue). As 3.4.4 hasn't been released yet, the current stubs are okay.

**Pull request merged**